packages/openapi-generator/src/options/options-per-service.ts

```diff
@@ -1,11 +1,21 @@
 import { existsSync, promises } from 'fs';
+import { parse } from 'path';
+import { EOL } from 'os';
+import { UniqueNameGenerator } from '@sap-cloud-sdk/util';
+import { getRelativePath } from '../generator';
+import { ParsedGeneratorOptions } from './generator-options';
 
 const { readFile } = promises;
 
 /**
  * Represents the service options for all services, mapped from the input file path to the service configuration.
  */
-export type OptionsPerService = Record<string, Partial<ServiceOptions>>;
+export type OptionsPerService = Record<string, ServiceOptions>;
+
+/**
+ * Partial OptionsPerService
+ */
+type PartialOptionsPerService = Record<string, Partial<ServiceOptions>>;
```

Comment:
I think this comment got lost and is on the wrong line:https://github.com/SAP/cloud-sdk-js/pull/1257/files#r623923247